src/Microsoft.AspNetCore.Mvc.Core/ModelBinding/Binders/BodyModelBinder.cs

```diff
@@ -89,12 +114,15 @@ public BodyModelBinder(IList<IInputFormatter> formatters, IHttpRequestStreamRead
 
             var httpContext = bindingContext.HttpContext;
 
+            var allowEmptyInput = _options?.AllowEmptyInputInBodyModelBinding == true;
```

Comment:
Not mandatory (of course) but the variable name should be more like the property.